app/views/content/life-as-a-teacher/explore-subjects/chemistry/_article.html.erb

```diff
@@ -1,46 +1,32 @@
-    <h2 class="heading--box-blue">Why teach chemistry</h2>
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
     <div class="inset">
-      <p>Chemistry underpins everything, so you'll be amazed at the questions your pupils will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no two days are the same in the classroom.</p>
-      <p>In addition to the personal rewards of teaching chemistry, there are practical benefits too, including:</p>
-      <ul>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">generous and secure pension</a></li>
-        <li>more holiday days than people in many other professions</li>
-        <li>opportunities for progression</li>
-      </ul>
+      <p><%= @front_matter["subject"].titleize %> underpins everything, so you'll be amazed at the questions your pupils will start asking. Leading hands-on lab experiments and creating engaging lessons will mean no two days are the same in the classroom.</p>
+      <!-- Why teach: benefits -->
+      <!--Example of alternative way to pass front matter or alernative text to the partial templates -->
+      <%= render 'content/life-as-a-teacher/explore-subjects/benefits', subject: @front_matter["subject"] %>
     </div>
   </section>
 
-  <section class="col col-720">
-    <%= render Content::QuoteComponent.new(
-    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
-    name: "Chris Farr, teacher training adviser",
-    large: true
-  ) %>
-  </section>
+  <!-- Quote: Chris Farr -->
+  <%= render 'content/general/quotes/quote_chris' %>
 
   <section class="col col-720 col-space-l-top">
     <h2 class="heading--box-blue">Shape your pupils' futures</h2>
     <div class="inset">
-      <p>Teaching chemistry opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from medicine and forensic science, to palaeontology and oceanography.</p>
-      <p>By solving challenging problems, your pupils could make significant contributions to these impactful areas.</p>
-      <p>Your pupils will learn more from you than you might think.  As their teacher, you’ll guide your pupils though the skills they need to show their understanding of chemistry. You'll teach them valuable life skills such as critical thinking, decision making and communication.</p>
+      <p>Teaching <%= @front_matter["subject"] %> opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from medicine and forensic science, to palaeontology and oceanography.</p>
```

Comment:
Don't think we need the [subject] pulling in for this as the second sentence is is different for each subject @sarahcrack

Reply:
no probs - reverted back to stating subject name in the code :)